README.md

```diff
@@ -1,22 +1,36 @@
 # Drone Manager
 
-This system is responsible for the following:
+This system is responsible for interfacing between the frontend systems and the drone management software provided by manufacturers.
+
+It takes areas of interest and resolves them into search patterns to photograph for the backend drone software.
+It also allows the frontend to get information about the drones' statuses.
 
 ## Running without docker
-```
+
+```bash
 python3 -m venv venv
-. venv/bin/activate (linux) or ./venv/Scripts/activate (win)
+. venv/bin/activate # linux
+./venv/Scripts/activate # windows
 python3 -m pip install -e
```

Comment:
I don't think we need this line, I don't know why I originally added it but maybe we should remove it 😅